WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -89,6 +89,15 @@ final class ShippingLabelFormViewModel {
         resultsController.fetchedObjects
     }
 
+    /// Check for international shipping based on whether destination country is other than US.
+    ///
+    var isInternationalShipping: Bool {
```

Comment:
Looking at the [web implementation](https://github.com/Automattic/woocommerce-services/blob/f772b6ed36fad5576582c9b8c5c044a6f0b7d2c2/client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js#L191), it seems like we should require a phone number for customs if the origin or destination is one of the US Military States.

Other than that, it's checking that the origin and destination country is different. I understand right now we only support shipping from the US, so I don't think there's a practical difference, but it might be more future-proof to have the equality check rather than `destinationCountriesWithoutPhoneRequirement`

Reply:
Thank you so much for pointing me to the web implementation! I really should look at that to double check my logic - I have duplicated the implementation for extra safety in fbb903f5ff31bcc4c5115e9eba9af6db3818a1eb.